implement countdown ui component

rezaerami opened the pull request:
add the countdown component
created countdown worker
moved countdown interval to the worker
added default template to domponent
added docz to the component
added test to cover 50% of branches for now

Commit:
implement countdown ui component

add the countdown component
created countdown worker
moved countdown interval to the worker
added default template to domponent
added docz to the component
added test to cover 50% of branches for now

sayjeyhi commented on package.json:
```diff
@@ -105,7 +105,8 @@
     "shelljs": "^0.8.3",
-    "webpack": "^4.41.0"
+    "webpack": "^4.41.0",
+    "worker-plugin": "^4.0.3"
```
There is no way to handle this with regular `Worker` or `rollup` based solution? we do not use `webpack` here

rezaerami replied:
unfortunately, the worker loader works with the Webpack file loader, I'm gonna check the comlink also, to see if it needs the Webpack file loader or not so I'm gonna refactor it with comlink, otherwise, it would be the only way

rezaerami replied:
as I checked the history of the package.json, the webpack had been added it the first commit!
besides, the comlink does not have any dependency to the Webpack, so it is possible to refactor, but the're no need to.

Commit:
Merge branch 'develop' of https://github.com/snappmarket/frontend-toolbox into feature/reza/offThreadTimer

Commit:
fix package.json

since we're using rollup and there's no need to have the webpack and
it's unused, removed it.
